src/main/asciidoc/inc/_goals.adoc

```diff
@@ -48,6 +48,10 @@ in the next sections.
 |Save images to a file
 |
 
+|**<<{plugin}:tag>>**
+|Tag images
+|install
```

Comment:
```suggestion
|Tags an image so that it becomes part of a repository.
```

Reply:
Improved description to be more detailed